triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Server.scala

```diff
@@ -511,34 +515,34 @@ object Server {
     implicit val esf: ExecutionSequencerFactory =
       new AkkaExecutionSequencerPool("TriggerService")(untypedSystem)
 
-    val authClient = authConfig match {
+    val authClientRoutes = authConfig match {
       case NoAuth => None
       case AuthMiddleware(uri) =>
-        Some(
+        val client =
           AuthClient(
             AuthClient.Config(
               authMiddlewareUri = uri,
               redirectToLogin = authRedirectToLogin,
-              callbackUri = authCallback.getOrElse {
-                Uri().withScheme("http").withAuthority(host, port).withPath(Path./("cb"))
-              },
               maxAuthCallbacks = maxAuthCallbacks,
               authCallbackTimeout = authCallbackTimeout,
               maxHttpEntityUploadSize = maxHttpEntityUploadSize,
               httpEntityUploadTimeout = httpEntityUploadTimeout,
             )
           )
-        )
+        val routes = client.routesAuto(authCallback.getOrElse(Uri().withPath(Path./("cb"))))
+        Some((client, routes))
     }
+    val authClient = authClientRoutes.map(_._1)
```

Comment:
I may be missing something but from the diff I can't see where this is used.

Reply:
It's used for token refresh [further below](https://github.com/digital-asset/daml/blob/41c31007a546767870bf6706d272687f2f942f01/triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Server.scala#L629).